src/Dropdown/Dropdown.tsx

```diff
@@ -80,8 +54,8 @@ export type DropdownProps = DefaultProps & TruncateProps
 export const Dropdown = forwardRef(function Dropdown(
   {
     id: idProp,
-    className = '',
     label,
+    className = '',
```

Comment:
is it necessary to default this to empty?

Reply:
I'm not 100% sure, but I didn't want to risk something silently failing somewhere, that we only then find out about when a user tells us - so I kept the existing pattern in this instance.